test/demangle-cases/bug-7521-powerpc.asm.json

```diff
@@ -124,11 +124,12 @@
     {
       "labels": [
         {
-          "name": "some_namespace::function()",
+          "name": "_ZN14some_namespace8functionEv@toc@ha",
```

Comment:
But this case has regressed now: While clicking still works (`target` tracks the correct target), because the `name` is no longer demangled, the `range` will be incorrect.

Reply:
Fixed by https://github.com/compiler-explorer/compiler-explorer/pull/7522/commits/f34a4296bb544d4901d81c00971e8a886fad9855.